src/modules/mavlink/mavlink_mission.cpp

```diff
@@ -1409,14 +1409,20 @@ MavlinkMissionManager::parse_mavlink_mission_item(const mavlink_mission_item_t *
 			}
 			break;
 
+		case MAV_CMD_DO_MOUNT_CONTROL:
+			mission_item->nav_cmd = (NAV_CMD)mavlink_mission_item->command;
+
+			// Transform from ENU to NED frame
+			mission_item->params[2] = -mavlink_mission_item->param3;
```

Comment:
Waiting for feedback on https://github.com/PX4/Firmware/issues/8599#issuecomment-355919393 to be sure this is the only axis that needs flipping.